utils/symbol-utils.stanza

```diff
@@ -135,6 +135,12 @@ public defn unit-rectangle (p0:Point|[Double, Double], p1:Point|[Double, Double]
 
 val MM-TO-POINT = 72.0 / 24.5
 
+public defn unit-text (l:String, p:Pose, txt, size:Double, a:Anchor, scale:Double) :
+  inside pcb-symbol :
+    ;val pt-size = to-int(ceil(size * scale * MM-TO-POINT)) [TODO: Fix this]
+    val pt-size = 0.7056 ;[Placeholder]
```

Comment:
I see you just copy pasted. Not great to leave such thing in ocdb which is public but it was already so okay